src/crimson/os/alienstore/alien_collection.h

```diff
@@ -14,12 +14,16 @@ namespace crimson::os {
 struct AlienCollection final : public FuturizedCollection {
 
   ObjectStore::CollectionHandle collection;
-
+public:
```

Comment:
nit, might want to define `AlienCollection` using `class` to make it explicit that `AlienCollection` is not a plain POD, and not all members are public.